src/main/java/com/aws/greengrass/util/platforms/StubResourceController.java

```diff
@@ -20,12 +23,12 @@ public void removeResourceController(GreengrassService component) {
 
     @Override
     public void updateResourceLimits(GreengrassService component, Map<String, Object> resourceLimit) {
```

Comment:
how often are these logs going to trigger? I don't want our log file flooded by these warnings which are not all that useful

Reply:
@shaguptashaikh

Reply:
Looks like it's only during startup and initial handling of run script, but yes, want to hear from @shaguptashaikh

Reply:
This log will be produced on each restart of a component that has been configured with system resource limits on a platform that doesn't support system resource limits.

Reply:
I don't know if there will be a way to not log it on each restart but still account for all cases like, customer puts resource limits on an already running component

Reply:
Sounds to me like this will log too much stuff.

Reply:
I don't think that adding these logs really adds much value at all. But if we really want it, then I'd suggest a one-shot sort of thing so that it only logs once per GG startup.

Reply:
Does our system know the difference between 'first time startup' and restart? Where is the right place to put a one time check?